app/src/main/java/com/example/kotlinplayground/StudentNameSantorini.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.kotlinplayground
+
+import kotlin.random.Random
+
+class StudentNameSantorini(numberOfStudentsInGroup: Int = 100) {
```

Comment:
Hi @yegorovmax Looks like overengineering. Can you please simplify this class and push one commit with all "all properties" and second teammate push commit with "all methods". Btw, you can take a look at the other PR.